Sources/TuistCore/Utils/OtoolController.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistSupport
+
+enum OtoolErrors: FatalError, Equatable {
+    case invalidOutput
+
+    var description: String {
+        switch self {
+        case .invalidOutput: return "The output is not in the format tuist expects"
+        }
+    }
+
+    var type: ErrorType {
+        switch self {
+        case .invalidOutput: return .abort
```

Comment:
If the output is invalid, it's a bug on our side because the contract between Tuist and `otool` is broken.